src/libraries/System.Text.Json/src/System/Text/Json/Document/JsonDocument.MetadataDb.cs

```diff
@@ -86,29 +86,34 @@ private struct MetadataDb : IDisposable
 
             internal int Length { get; private set; }
             private byte[] _data;
-#if DEBUG
-            private readonly bool _isLocked;
-#endif
+            private bool _convertToAlloc; // Convert the rented data to an alloc when complete.
+            private bool _isLocked; // Is the array the correct fixed size.
```

Comment:
What does `_isLocked` mean, now? Maybe we should rename it since its use, the comment, and name no longer match.

Reply:
It means basically the same as it did before -- the array is the correct size and thus shouldn't be increased in size.